cmd/podman/umount.go

```diff
@@ -24,18 +38,70 @@ func umountCmd(c *cli.Context) error {
 	}
 	defer runtime.Shutdown(false)
 
+	umountAll := c.Bool("all")
 	args := c.Args()
-	if len(args) == 0 {
+	if len(args) == 0 && !umountAll {
 		return errors.Errorf("container ID must be specified")
 	}
-	if len(args) > 1 {
-		return errors.Errorf("too many arguments specified")
+	if len(args) > 0 && umountAll {
+		return errors.Errorf("when using the --all switch, you may not pass any container IDs")
 	}
 
-	ctr, err := runtime.LookupContainer(args[0])
-	if err != nil {
-		return errors.Wrapf(err, "error looking up container %q", args[0])
+	umountContainerErrStr := "error unmounting container"
+	var lastError error
+	if len(args) > 0 {
+		for _, name := range args {
+			ctr, err := runtime.LookupContainer(name)
+			if err != nil {
+				if lastError != nil {
+					logrus.Error(lastError)
+				}
+				lastError = utils.WriteError(os.Stderr, errors.Wrapf(err, "%s %s", umountContainerErrStr, name), lastError)
+				continue
+			}
+
+			if err = unmountContainer(ctr); err != nil {
+				if lastError != nil {
+					logrus.Error(lastError)
+				}
+				lastError = utils.WriteError(os.Stderr, errors.Wrapf(err, "%s %q", umountContainerErrStr, name), lastError)
```

Comment:
I Believe these are wrong and cut and pasted from the buildah pr, which has the same issues.

Reply:
The utils.WriteError call should not be there.  Won't this output to stderr right away.  We should just say 
lastError = errors.Wrapf(err, ...)

Reply:
Cut and pasted from the Buildah PR indeed, but I think these are OK as is.  However, I just backtracked on removing the logrus calls, I'm leaving them and dropping this utils.WriteError() and just setting lastError to errors.Wrapf as suggested by @rhatdan.